appserver/tests/appserv-tests/devtests/cdi/osgi-cdi/simple-wab-with-cdi/WebTest.java

```diff
@@ -51,8 +50,8 @@ public static void main(String[] args) {
     }
 
     public void doTest() {
-        try { 
-            invoke();
+        try {
+            // invoke();
```

Comment:
Why's this one commented out?

Reply:
That's the OSGi testing that would fail because fighter fish is not there. WABs don't deploy correctly at the moment, but I've also disabled the test at a higher level, so we might uncomment that now.